changelog.md

```diff
@@ -0,0 +1,59 @@
+# changelog
+
+## 1.0.0-rc1
+
+This is the first release targeting Qubes 4.2 and will require
+a full reinstall. As we have reached 1.0.0, we are transitioning
+from a closed pilot to an open beta program. If you are interested
+in using SecureDrop Workstation, please reach out to us at TBD.
+
+* Add Qubes 4.2 support (#970, #976, #983, #987, #993, #995)
+* Create base template by cloning debian-12-minimal template (#970)
+* Switch to v2 qubes-rpc policies (#990)
```

Comment:
I think for clarity, we shouldn't call this "v2", since the v2 qrexec service is deprecated and we're on qrexec v3. (The .policy files are documented in the v3 page but not explicitly referred as "v3 policy files", I just see "[the new format](https://github.com/QubesOS/qubes-issues/issues/4370)").  I'd say either "Switch to new RPC .policy file format" and link https://www.qubes-os.org/doc/qrexec/#policy-files andn/or go with "qrexec3-compatible .policy files"

Reply:
Ack, went with your first suggestion.